src/lib/net/util/server.ts

```diff
@@ -0,0 +1,107 @@
+import { Players, RunService } from "@rbxts/services";
+
+import { FunctionUtil } from "../../util/functions";
+import { Identifiers } from "./identifiers";
+import { Contents, InitialPacket, Packet, ServerListener } from "./types";
+
+function getWhiteListed(list: { [x: string]: boolean }): Array<Player> {
+	const to: Array<Player> = [];
+
+	for (const player of Players.GetPlayers()) {
+		if (list[player.Name] !== undefined) {
```

Comment:
This checks if the player exists as a key on the `list`, but doesn't check if their corresponding value in `list` is true. Is this intentional?

Reply:
Yes, it is intentional.

Reply:
Yes, it is intentional.